### ISSUE TITLE
Add a volume slider

### ISSUE THREAD

**Issue by DrabWeb** [ttencate/jfxr#24]
A volume slider would be very nice, because the sounds are really loud when my computers at regular volume

**Comment by blurymind** [comment]
I used this after using jsfx, and the sound is already quiet enough imo. It is softer and quieted than what jsfx produces

**Comment by pfroud** [comment]
The Amplification slider in the Output section can be used to lower the volume:

![image](https://github.com/ttencate/jfxr/assets/9056906/fe24ef96-39df-4407-b67a-68ca76ea690f)

**Comment by ttencate** [comment]
Yes, but that gets applied to the resulting file too. This request, as I understand it, is about playback volume only.
